float32 x
float32 y
float32 z
float32 pitch
float32 yaw
---
map_block_msg[] visible_blocks
